baseplate/lib/retry.py

```diff
@@ -124,14 +126,15 @@ def __init__(self, policy: RetryPolicy, base: float):
         self.subpolicy = policy
         self.base = base
 
-    def yield_attempts(self) -> Iterator[Optional[float]]:
-        for attempt, time_remaining in enumerate(self.subpolicy):
+    def yield_attempts(self) -> Iterator[Tuple["RetryPolicy", Optional[float]]]:
+        for attempt, (return_policy, value) in enumerate(self.subpolicy):
```

Comment:
nit: this probably should be `retry_policy` instead of `return_policy`? the name "return_policy" sounds weird.

Reply:
ah, i meant to say `retry_policy` here oops